### PR TITLE
Sound notification

### PR DESCRIPTION
**Pull request by lemieux**
@dannytranlx @Mario54

### CODE REVIEW

File: source/includes/customization/_sound_notification_js.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+## Sound notification
+
+By default, a sound notification will be played when a new message comes in and the window in not in focus.
```
Comment (Mario54) [inline source/includes/customization/_sound_notification_js.md]: `and the window *is* not in focus`

File: source/includes/customization/_sound_notification_js.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+To disable this feature, you need add the `soundNotificationEnabled` option to the `Smooch.init` call, like this:
```
Comment (Mario54) [inline source/includes/customization/_sound_notification_js.md]: Should this be an option in the Web SDK integration page?

**Comment by Mario54** [comment]
Rest is 👍

File: source/includes/customization/_sound_notification_js.md (hunk 2 above)
Comment (lemieux) [inline source/includes/customization/_sound_notification_js.md]: I talked with jp about that yesterday and I think we might port all init options to the integration page yeah. But not right now.

File: source/includes/customization/_sound_notification_js.md (hunk 1 above)
Comment (lemieux) [inline source/includes/customization/_sound_notification_js.md]: good catch

File: source/includes/customization/_sound_notification_js.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+```js
+Smooch.init({
+    appToken: 'your_app_token',
+    givenName: 'Cool',
+    surname: 'Person',
+    soundNotificationEnabled: false, // Add this line to your 'Smooch.init' call
```
Comment (mspensieri) [inline source/includes/customization/_sound_notification_js.md]: nit: trailing comma

**Comment by mspensieri** [comment]
:+1:

**Pull request merged**